datacite/schemas/datacite-v4.1.json

```diff
@@ -0,0 +1,700 @@
+              "funderIdentifierType"
+            ]
+          },
+          "awardNumber": {
+            "description": "The code assigned by the funder to a sponsored award (grant).",
+            "type": "object",
+            "properties": {
+              "awardNumber": {
+                "type": "string"
+              },
+              "awardURI": {
+                "$ref": "#/definitions/uri"
+              }
+            },
+            "additionalProperties": false,
+            "required": ["awardNumber"]
+          },
+          "awardTitle": {
+            "description": "The human readable title of the award (grant).",
+            "type": "string"
+          }
+        },
+        "additionalProperties": false,
+        "required": ["funderName"]
+      }
+    },
+    "geoLocations": {
+      "description": "Spatial region or named place where the data was gathered or about which the data is focused.",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "geoLocationPoint": {
+            "description": "A point location in space.",
+            "type": "object",
+            "properties": {
+              "pointLongitude": {
+                "description": "Longitudinal dimension of point.",
+                "type": "number"
+              },
+              "pointLatitude": {
+                "description": "Latitudinal dimension of point.",
+                "type": "number"
+              }
+            },
+            "additionalProperties": false,
+            "required": [
+              "pointLongitude",
+              "pointLatitude"
+            ]
+          },
+          "geoLocationBox": {
+            "description": "The spatial limits of a box.",
+            "type": "object",
+            "properties": {
+              "westBoundLongitude": {
+                "description": "Western longitudinal dimension of box.",
+                "type": "number"
+              },
+              "eastBoundLongitude": {
+                "description": "Eastern longitudinal dimension of box.",
+                "type": "number"
+              },
+              "southBoundLatitude": {
+                "description": "Southern latitudinal dimension of box.",
+                "type": "number"
+              },
+              "northBoundLatitude": {
+                "description": "Northern latitudinal dimension of box.",
+                "type": "number"
+              }
+            },
+            "additionalProperties": false,
+            "required": [
+              "westBoundLongitude",
+              "eastBoundLongitude",
+              "southBoundLatitude",
+              "northBoundLatitude"
+            ]
+          },
+          "geoLocationPlace": {
+            "description": "Spatial region or named place where the data was gathered or about which the data is focused.",
+            "type": "string"
+          },
+          "geoLocationPolygon": {
```

Comment:
@lnielsen I guess for similar reasons, we will support ``geoLocationPolygon`` alongside ``geoLocationPolygons`` until Kernel 5?